mmv1/products/networksecurity/FirewallEndpoint.yaml

```diff
@@ -110,3 +114,8 @@ properties:
     name: 'state'
     description: The current state of the endpoint.
     output: true
+  - !ruby/object:Api::Type::String
+    name: 'billingProjectId'
```

Comment:
Will not providing the billing project make the resource creation failed in all cases?
Adding a required field is a breaking change that we'd wait until the next major release, unless without the field specified, all current configurations will fail. See details at https://googlecloudplatform.github.io/magic-modules/develop/breaking-changes/breaking-changes/#field-level-breaking-changes and https://googlecloudplatform.github.io/magic-modules/develop/breaking-changes/make-a-breaking-change/#in-minor-releases

Reply:
Hey @shuyama1, thanks for the quick feedback. This breaks all tests cases according to [this](https://github.com/hashicorp/terraform-provider-google/issues/17396).
Unfortunately, this field was introduced as a required field after the enhacement PR was merged.

Reply:
Would you mind updating the test to include this field?

Reply:
Hello @shuyama1 I added the field to the basic test so it fixes it.